domain/modelagent/errors/errors.go

```diff
@@ -9,4 +9,18 @@ const (
 	// AgentVersionNotFound describes an error that occurs
 	// when an agent version record is not present.
 	AgentVersionNotFound = errors.ConstError("agent version not found")
+
+	// MachineAgentVersionNotSet describes an error that occurs when a machine
+	// does not have its agent version set.
+	MachineAgentVersionNotSet = errors.ConstError("machine agent version not set")
+
+	// MissingAgentBinaries describes an error that occurs when agent binaries
+	// are missing for a given entity that runs agent binaries within the
+	// model, eg units and machines. When agent binaries are missing, it
+	// means that the model does not have a copy of the binaries.
+	MissingAgentBinaries = errors.ConstError("missing agent binaries")
+
+	// UnitAgentVersionNotSet describes an error that occurs when a unit
+	// does not have its agent version set.
+	UnitAgentVersionNotSet = errors.ConstError("unit agent version not set")
```

Comment:
Can we move these two NotSet errors together?

Reply:
Yeah, we already have a single agent binary missing error.